specs/vep-554-apache-airflow-integration/README.MD

```diff
@@ -395,6 +402,31 @@ Secrets will be stored as part of the Airflow Connection like follows:
 | connection.login    | Client ID     |
 | connection.password | Client Secret |
 
+For the connection itself, a standard Airflow Connection object would be used,
+
+```python
+Connection(
+  conn_id="",
+  conn_type="vdk",
+  description=None,
+  login="<client-id-or-empty>",
+  password="<password-or-oauth-token>",
+  host="<Control-Service-host-URL>",
+  port=,
+  schema=None,
+  extra=json.dumps(dict(auth_endpoint="<oauth2-authentication-endpoint>"))
```

Comment:
what is auth endpoint ? There are a couple of options.

Reply:
Fixed, thanks